twitter-langchain/examples/chatbot/.env-local

```diff
@@ -0,0 +1,5 @@
+OPENAI_API_KEY=
+TWITTER_ACCESS_TOKEN=
+TWITTER_ACCESS_TOKEN_SECRET=
+TWITTER_API_KEY=
```

Comment:
the bearer token is also needed?

Reply:
this is an oddity that i dug into. there is a discrepancy between the libs being used in agentkit.py and agentkit.js. the python twitter library will not get account mentions without the bearer token provided however, the nodejs lib is happy with just the app and token keys and secrets. 

the bearer token is used for twitter-api-v2 integration. i think the nodejs libary is either calling the v1 mentions, or is obtaining a v2 token behind the scenes.

was thinking that maybe we could try another twitter library for agentkit.py, or maybe patch twit to remove the bearer token dep?